This is a basic list of what needs to be done for the project and how we might do that

This portion is based on the idea of finding what genomes have a gene and what that gene does

---

## Helper functions

- [ ] Precollect & Curate Reference Genomes (genbank)

- [ ] genbank -> fasta (nucleic acids) function

- [ ] genbank -> fasta (amino acids) function

- [ ] Convert fasta to BLAST DB function

- [ ] Create local rps BLAST DB  function

- [ ] Create new BLAST DB function (protien and nucleic acids)

- [ ] Create new rps BLAST DB function 

- [ ] Create new psi BLAST DB function

- [ ] Create aligning function which extracts 

## Report Class

- [ ] Input seqs, if more than one input align to determine cutoffs identity and coverage cutoffs otherwise have user define

- [ ] BLAST sequences against appropriate databases and return based on cutoffs

- [ ] Extract gene function info (domains, eggnog info, EC number, goterms etc....)
  
  - [ ] Compile blast hits of similar info and lay out distrabution 

- [ ] create curated list of genomes with meta info 



## Meta Report Class

- [ ] Well worry about it later
